components/board/pi/impl/i2c.go

```diff
@@ -102,7 +86,8 @@ func (s *piPigpioI2CHandle) ReadBlockData(ctx context.Context, register byte, nu
 	return data, nil
 }
 
-func (s *piPigpioI2CHandle) WriteBlockData(ctx context.Context, register byte, numBytes uint8, data []byte) error {
+func (s *piPigpioI2CHandle) WriteBlockData(ctx context.Context, register byte, data []byte) error {
+	numBytes := len(data)
```

Comment:
[test-req] i2c device on pi.

Reply:
This function is called literally nowhere in our codebase. but I can temporarily rewrite an accelerometer driver to use `WriteBlockData` with a block of length 1, rather than just `WriteByteData`...

Reply:
We should use our logic analyzers with simple scripts for board functionality tests in the future. It's easier. However, thanks for figuring that out with the adxl!

Maybe also buy some for our team.

Reply:
Yup, that worked! See the two most recent commits (one making the change to call this function, and one to revert it): the behavior of the MPU345 accelerometer plugged into a raspberry pi was identical.

Reply:
> Maybe also buy some for our team.

There are a bunch of MPU6050's in the lab, but you need to solder header pins on yourself.

Reply:
I meant logic analyzers, not mpus